### ISSUE TITLE
False positive for "FooUpToBar"

### ISSUE THREAD

**Issue by myint** [lyda/misspell-check#22]
Somehow, the current camel-case splitting regular expression is not adequate.

``` bash
$ echo 'FooUpToBar' > test
$ misspellings test
test:1: UpTo -> "Up to"
```

"UpTo" is fine though.

``` bash
$ echo 'UpTo' > test
$ misspellings test
```

Testing the above reminded me that it would be nice to have standard in support (`misspellings -`).
